### DISCUSSION TITLE
anyone have a model for bile acids already developed?

### DISCUSSION THREAD

**Discussion by ssalerno1** [Open-Systems-Pharmacology/Forum#1285] [Questions & Problems]

**Comment by Yuri05** [comment]
moving this to the forum

**Comment by StephanSchaller** [comment]
@ssalerno1, (Just so we also have an answer on this Forum) Yes, there is / we do:

https://pubmed.ncbi.nlm.nih.gov/31611804/
https://ascpt.onlinelibrary.wiley.com/doi/full/10.1002/cpt.2406
https://www.biorxiv.org/content/10.1101/450965v1
